.github/workflows/dev-publishing.yml

```diff
@@ -51,17 +59,15 @@ jobs:
       - name: Run tests
         run: npm run test:without-coverage
 
-      - name: Read package.json
-        uses: culshaw/read-package-node-version-actions@v1
-        id: package
-
       - name: Commit and tag the updated version
-        uses: EndBug/add-and-commit@v7
-        with:
-          message: 'ci: bump version to ${{ steps.package.outputs.version }}'
-          tag: 'v${{ steps.package.outputs.version }}'
-          author_name: ${{ env.GITHUB_USER }}
-          author_email: ${{ env.GITHUB_EMAIL }}
+        run: |
+          git config user.name ${{ env.GITHUB_USER }}
+          git config user.email ${{ env.GITHUB_EMAIL }}
+          git add .
+          git commit -m "ci: bump version to ${{ steps.package.outputs.version }}"
+          git push
```

Comment:
Now I see this line as redundant, as `git push --tags` on L70 does pushing commits and tags alike.

Reply:
`git push` do not push the tags, this is why we need then to do a `git push --tag` specifically for the tags.

Reply:
Aargh, right. I use `git push --follow-tags`, but I use annotated tags only (it does not push lightweight tags).

But according [this SO answer](https://stackoverflow.com/a/3745250/3408342), there are several options. Lastest update suggests to use the following command, which should push lightweight and annotated tags alike (`git` 2.4.x is needed):

```bash
git push --atomic origin dev v${{ steps.package.outputs.version }}
```

Reply:
Thank you! I'll try to have a closer look on that in the next days!